lib/puppet/type/cisco_bgp_af.rb

```diff
@@ -196,6 +198,34 @@ def self.title_patterns
     newvalues(:true, :false, :default)
   end # property :default_information_originate
 
+  newproperty(:maximum_paths) do
+    desc "Configures the maximum number of equal-cost paths for load sharing.
+          Valid values are integers between 1 and 64, default value is 1."
```

Comment:
/Valid values are integers between 1 and 64/Valid value is an integer in the range 1 - 64/